### ISSUE TITLE
Email ledger admin when new accounts are created

### ISSUE THREAD

**Issue by emschwartz** [interledgerjs/ilp-kit#70]
It would be nice to get notifications when people sign up for your ledger / ilp-kit

**Comment by Winthan** [comment]
That would be great, right now I am using with mixpanel for the tracking.

**Comment by vhpoet** [comment]
@Winthan there's a `Users` page for the admin where you can see all of the ledger users with their balances.

**Comment by Winthan** [comment]
@vhpoet Yes, I saw that. It is so simple for now. I am tracking with Mixpanel for my custom tracking. 

And I think you may wanna add some options for admin to manage to account such as freeze, close, and notify account, so on.

**Comment by vhpoet** [comment]
@Winthan yep, that's just the start :)
